src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -21,127 +72,1329 @@ const ActorInit En_Po_Sisters_InitVars = {
+            this->unk_199 |= 1;
+            this->collider.body.bumper.flags = 0x4FC7FFEA;
+            if (this->unk_19A != 0) {
+                this->unk_19A--;
+            }
+            if (this->unk_19A == 0) {
+                this->unk_197 = 20;
+                func_80AD93C4(this);
+            }
+        } else {
+            func_80AD9F1C(this);
+        }
+    } else {
+        this->unk_22E.a = (this->skelAnime.animCurrentFrame * 255.0f) / this->skelAnime.animFrameCount;
+        if (this->unk_194 == 0) {
+            func_80AD97C8(this, globalCtx);
+        }
+    }
+}
+
+void func_80ADAFC0(EnPoSisters* this, GlobalContext* globalCtx) {
+    s32 i;
+
+    this->unk_19A++;
+    this->unk_198 = CLAMP_MAX(this->unk_198 + 1,8);
+    for (i = this->unk_198 - 1; i > 0; i--) {
+        this->unk_234[i] = this->unk_234[i - 1];
+    }
+    this->unk_234[0].x =
+        (Math_Sins((this->actor.shape.rot.y + this->unk_19A * 0x3000) - 0x4000) * (3000.0f * this->actor.scale.x)) +
+        this->actor.posRot.pos.x;
+    this->unk_234[0].z =
+        (Math_Coss((this->actor.shape.rot.y + this->unk_19A * 0x3000) - 0x4000) * (3000.0f * this->actor.scale.x)) +
+        this->actor.posRot.pos.z;
+    if (this->unk_19A < 8) {
+        this->unk_234[0].y = this->unk_234[1].y - 9.0f;
+    } else {
+        this->unk_234[0].y = this->unk_234[1].y + 2.0f;
+        if (this->unk_19A >= 16) {
+            if (Math_ApproxF(&this->actor.scale.x, 0.0f, 0.001f) != 0) {
+                func_80AD9A54(this, globalCtx);
+            }
+            this->actor.scale.z = this->actor.scale.x;
+            this->actor.scale.y = this->actor.scale.x;
+        }
+    }
+    if (this->unk_19A == 16) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_PO_DEAD2);
+    }
+}
+
+void func_80ADB17C(EnPoSisters* this, GlobalContext* globalCtx) {
+    Vec3s* vec;
+
+    this->unk_19A++;
+    if (this->unk_19A == 64) {
+        Flags_SetSwitch(globalCtx, this->actor.params);
+        Audio_PlaySoundAtPosition(globalCtx, &this->actor.posRot.pos, 30, NA_SE_EV_FLAME_IGNITION);
+        if (this->unk_194 == 0) {
+            Flags_UnsetSwitch(globalCtx, 0x1B);
+        }
+        globalCtx->envCtx.unk_BF = 0xFF;
+        func_80078884(NA_SE_SY_CORRECT_CHIME);
+        Actor_Kill(&this->actor);
+    } else if (this->unk_19A < 32) {
+        func_80AD9240(this, this->unk_19A, &this->actor.posRot.pos);
+    } else {
+        func_80AD9240(this, 64 - this->unk_19A, &this->actor.posRot.pos);
+    }
+    if (this->unk_19A == 32) {
+        vec = &D_80ADD7A4[this->unk_194];
+        this->actor.posRot.pos.x = vec->x;
+        this->actor.posRot.pos.y = vec->y;
+        this->actor.posRot.pos.z = vec->z;
+    }
+}
+
+void func_80ADB2B8(EnPoSisters* this, GlobalContext* globalCtx) {
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    if (this->actor.xzDistFromLink < 130.0f) {
+        func_80AD9DF0(this, globalCtx);
+    }
+    if (func_800A56C8(&this->skelAnime, 0.0f) != 0) {
```

Comment:
i believe this one is a bool as well, and just lets you know if an animation has finished. going off memory though